### ISSUE TITLE
Dummy Apps' made when linting podspec's have xcodebuild warnings

### ISSUE THREAD

**Issue by AnthonyWharton** [CocoaPods/CocoaPods#9895]
* [X] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

Run `pod lib lint --verbose --sources=git@github.com:OrganisationAccount/CocoaPodSpecs.git --private`

Note that this is a private pod that I have renamed `MyCocoaPod`, with one private dependecy renamed `MyDependency`, all hosted under a private specs repo with a placeholder link as shown above.

## What did you expect to happen?

It to build successfully, with a few known notes (my comments prefixed with a `#`):

```
 -> MyCocoaPod (X.X.X)
# Covered by `--private`:
    - WARN  | source: Git SSH URLs will NOT work for people behind firewalls configured to only allow HTTP, therefore HTTPS is preferred.
# Harmless Background noise that is being filtered out as it is a note from XCode:
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | xcodebuild:  note: Building targets in parallel
    - NOTE  | xcodebuild:  note: Planning build
    - NOTE  | xcodebuild:  note: Constructing build description
    - NOTE  | xcodebuild:  note: Using legacy build system
# Seems to be a common[1] harmless[2], issue[3], I am not sure how to fix this:
    - NOTE  | xcodebuild:  note: Execution policy exception registration failed and was skipped: Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted" (in target 'MyDependency' from project 'Pods')
    - NOTE  | xcodebuild:  note: Execution policy exception registration failed and was skipped: Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted" (in target 'MyCocoaPod' from project 'Pods')
    - NOTE  | xcodebuild:  note: Execution policy exception registration failed and was skipped: Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted" (in target 'Pods-App' from project 'Pods')
    - NOTE  | xcodebuild:  note: Execution policy exception registration failed and was skipped: Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted" (in target 'App' from project 'App')
# Another common harmless issue in the lint process, although it would be nice if this could be addressed:
    - NOTE  | [iOS] xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')
```

[[1]](https://developer.apple.com/forums/thread/124529), [[2]](https://stackoverflow.com/questions/58750471/why-does-my-podspec-throws-multiple-execution-policy-exception-registration-fa), [[3]](https://stackoverflow.com/questions/58686708/execution-policy-exception-registration-failed-and-was-skipped-error-domain-nsp)

## What happened instead?

I also got the following note:

```
    - NOTE  | [OSX] xcodebuild:  warning: No App Category is set for target 'App'. Set a category by using the General tab for your target, or by adding an appropriate LSApplicationCategory value to your Info.plist. (in target 'App' from project 'App')
```

I have never seen this before, but I assume it is similarly harmless to the iOS code signing warning.  It would be nice if both of this warnings could be removed by creating an appropriate `Info.plist` file in the dummy app created when linting? This would both address the iOS code signing issue as well as the new macOS "No App Category is set" issue.

### Stack

```
   CocoaPods : 1.9.3
        Ruby : ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]
    RubyGems : 3.0.3
        Host : Mac OS X 10.15.4 (19E287)
       Xcode : 11.5 (11E608c)
         Git : git version 2.24.3 (Apple Git-128)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib
Repositories : master - git - https://github.com/CocoaPods/Specs.git @ 128d8bd316745e0a8b3a6cf5544e47140d05d502

               trunk - CDN - https://cdn.cocoapods.org/
               OrganisationAccount - git - git@github.com:OrganisationAccount/CocoaPodSpecs.git @ 7423656034a6ad202efa8e260ba2c558debace45

```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.4.1
cocoapods-try         : 1.1.0
```

## Project that demonstrates the issue

Unfortunately I cannot disclose this codebase, however when I have time I will try to reproduce this in a public project.

**Comment by dnkoutso** [comment]
Sure i think we can make this issue about fixing warnings for the default project `lint` creates.

**Comment by AnthonyWharton** [comment]
Thanks, please let me know if there is anything that I can do. I'm not terribly familiar with Ruby so from an implementation perspective perhaps am less useful, but I could play about with a clean project at some point and work out settings that perhaps should be changed in order to squash some of these warnings.

**Comment by dnkoutso** [comment]
This is fixed so far for 1.11.0 actually. Closing for now.

**Comment by dnkoutso** [comment]
This is the PR that fixed it https://github.com/CocoaPods/CocoaPods/pull/10787
